### ISSUE TITLE
Yanki not syncing

### ISSUE THREAD

**Issue by aesculapa** [kitschpatrol/yanki-obsidian#11]
Dear Yanki Plugin Developers,

I hope this message finds you well. I have been using the Yanki plugin to sync my Obsidian notes with Anki, and it has been incredibly useful. However, recently, the sync function has stopped working.

I have gone through several troubleshooting steps, including:

Verifying that the Yanki plugin is enabled and correctly configured in Obsidian.
Ensuring that AnkiConnect is installed and running properly within Anki.
Testing the AnkiConnect API to confirm it’s responding.
Double-checking the note format and Yanki settings to ensure they match the required configurations.
Reviewing the console logs in Obsidian for any related errors.
Checking for updates for both Yanki and AnkiConnect, and ensuring both are up to date.
Restarting both Obsidian and Anki, as well as reinstalling the Yanki plugin.
Turning off all the other Obsidian plugins.

I created a new folder for testing syncing - it seems anki sees it, however I keep receive an error message: "Write failed. File not found.". 
However my main folder with yanki notes is not synced at all.

I would greatly appreciate any assistance or guidance you could provide to help resolve this issue.

Thank you in advance for your support.

Best regards,
Aneta

**Comment by aesculapa** [comment]
After removing special symbols (ń, ł, ę, ą) I was finally able to sync the new folder. 
The issue with my default folder however persists...

**Comment by kitschpatrol** [comment]
Hi, thank you. Let's solve this, but I need some more info:

1. What platform are you on? Mac? Windows? Linux?

2. When deleting special characters partially fixed the issue, were those characters in the note file names or in the content of the notes?

3. Can I assume you're using the latest versions of Anki, Anki-Connect, Obsidian, and Yanki?

4. Do you remember when / which Yanki version update broke syncing for you?

5. Are you willing to share the folder of notes you're trying to sync so I can try to reproduce the issue? If so, please Zip the folder and upload to this Issue thread, or email it to me as an attachment at eric@ericmika.com.

Thank you.

**Comment by aesculapa** [comment]
Hi! Thank you for your response! 
1. I'm on Mac
2. Those were note file names automatically created from the prompt
3. The latest version, I did the update before it stopped syncing.
4. Sent

I noticed one more thing today. I was creating new notes within the folder using a hotkey. Those notes were created within the "new" folder. And it seems I have the same problem again. I receive the response "anki sync starting..." and then nothing.

**Comment by kitschpatrol** [comment]
Hi again, thanks for providing the test files.

So far, I'm able to reproduce the issue as you've described it in the yanki-obsidian plugin with the provided notes... but interestingly all of your notes sync fine via the [command-line version of yanki](https://github.com/kitschpatrol/yanki). So it's something Obsidian-specific. 🤔

I will update here when it's figured out, could be a day or two. Thanks for your patience.

**Comment by kitschpatrol** [comment]
Seems like an issue with embedded block wiki-links...

For a quick temporary fix, delete instances like this from your notes:

```markdown
![[Some Note#^5a90e0]]
```

Working on an actual fix.

**Comment by kitschpatrol** [comment]
This should be fixed in the just-released version 1.4.1. Please give it a try and let me know if it resolves your syncing issue.

For what it's worth, there shouldn't be any issues with Unicode characters like ń, ł, ę, ą, etc, that might have been a coincidence related to notes with embedded Markdown blocks. If you _are_ seeing character-related syncing failures, please open a new issue with an example.

**Comment by kitschpatrol** [comment]
I'm going to close this issue since it's fixed with the test materials you provided, but if you're still having issues please feel free to reopen.

**Comment by aesculapa** [comment]
Thank you very much for your help!

I was able to properly check it only today and it works. In terms of special characters I still receive a notification of an error however it doesn't block syncing other cards.

I appreciate your hard work!

**Comment by kitschpatrol** [comment]
Thank you for confirming!

Can you provide more details on the special characters issue? 

I'm not currently seeing character-related errors with the files you provided, and I'd like to reproduce / fix whatever's going on with that as well.

**Comment by aesculapa** [comment]
I think the ones I've sent you I might have fixed that.

Today I added a card which went like that:
"
Ułożenie znacznika w USG

—-
doglowowo w strzałkowym;
w prawo w czołowym
(w lewo na plecach)
"

I changed it to the one below and I stopped receiving an error.

"
Ulozenie znacznika w USG

—-
doglowowo w strzałkowym;
w prawo w czołowym
(w lewo na plecach)
"

Of you won’t be able to reproduce the error I will send you the file tomorrow.
Best,
